frontend/src/app/collab/collab.component.ts

```diff
@@ -102,13 +118,7 @@ export class CollabComponent implements OnInit, AfterViewInit {
       console.log(request)
       console.log(request[0] == 1)
       if (request[0] == 1) {
```

Comment:
I don't believe a request is reemitted back to the requester; I believe a change could be made to the service in order to avoid updating the observable unnecessarily (removing the `.next` on emitting a request). Additionally, I don't think the 0/1 value in `request[0]` is necessary: removing it would reduce possible issues arising with the double updates to the observable, which I believe is non-deterministic given there are no guarantees which update to the observable happens first